plane/schema/migrations/20240209001513_add-drone-pool-column.sql

```diff
@@ -0,0 +1,6 @@
+-- Add pool column to drone table
+
+alter table drone add column pool text not null default '';
```

Comment:
we should probably enforce a limit on this string, something like a `varchar(32)` would be reasonable.

Reply:
Fixed, went with `varchar(255)` since that was what the rest of the schema used, including for user-defined inputs.